test/prepare.sh

```diff
@@ -2,18 +2,39 @@
 
 set -eux -o pipefail
 
-IMG="${IMG:-controller:test}"
+IMG="${IMG:-localhost/controller:test}"
 LOGDIR="${LOGDIR:-/tmp/logs}"
+CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-}"
 
 mkdir -p "${LOGDIR}"
 
+if [ -z "${CONTAINER_RUNTIME}" ]; then
+    if podman version > /dev/null 2>&1;  then
```

Comment:
```suggestion
    if command -v podman &> /dev/null;  then
```
More standard way and also using `&>` as its simpler.